aten/src/ATen/native/quantized/Copy.cpp

```diff
@@ -17,6 +17,9 @@ Tensor& quantized_copy_from_float_(Tensor& self, const Tensor& src) {
   TORCH_CHECK(
       self.sizes().equals(src.sizes()),
       "Quantized copy only works with Tensors with the same shape");
+  TORCH_CHECK(
+      self.device() == kCPU,
+      "Quantized copy only works with QuantizedCPU Tensors");
```

Comment:
I'd suggest renaming this function to have `cpu` in the name to make it clearer it works for cpu only

Reply:
Good point, fixed.